docs/develop/financial-accounting.md

```diff
@@ -0,0 +1,209 @@
+---
+sidebar_position: 2
+---
+
+# Financial Accounting
+
+For developers with non-financial backgrounds, TigerBeetle's use of accounting concepts like debits
+and credits may be one of the trickier parts to understand. However, these concepts have been the
+language of business for hundreds of years, so we promise it's worth it!
+
+This page goes a bit deeper into debits and credits, double-entry bookkeeping, and how to think
+about your accounts as part of a type system.
+
+## Building Intuition with Two Simple Examples
+
+If you have an outstanding loan and owe a bank $100, is your balance $100 or -$100? Conversely, if
+you have $200 in your bank account, is the balance $200 or -$200?
+
+Thinking about these two examples, we can start to build an intuition that the **positive or
+negative sign of the balance depends on whose perspective we're looking from**. That $100 you owe
+the bank represents a "bad" thing for you, but a "good" thing for the bank. We might think about
+that same debt differently if we're doing your accounting or the bank's.
+
+These examples also hint at the **different types of accounts**. We probably want to think about a
+debt as having the opposite "sign" as the funds in your your bank account. At the same time, the
+types of these accounts look different depending on whether you are considering them from the
+perspective of you or the bank.
+
+Now, back to our original questions: is the loan balance $100 or -$100 and is the bank account
+balance $200 or -$200? On some level, this feels a bit arbitrary.
+
+Wouldn't it be nice if there were some **commonly agreed-upon standards** so we wouldn't have to
+make such an arbitrary decision? Yes! This is exactly what debits and credits and the financial
+accounting type system provide.
+
+## Types of Accounts
+
+In financial accounting, there are 5 main types of accounts:
+
+- **Asset** - what you own, or "a present economic resource controlled by the entity as a result of
+  past events. An economic resource is a right that has the potential to produce economic
+  benefits."[^1]
```

Comment:
Asset - what you own, can produce income or can sell. (okay to be a little loose in the name of understanding—we don't need the IFRS reference here)